tuf/repository_tool.py

```diff
@@ -2174,6 +2174,51 @@ def get_delegated_rolenames(self):
 
 
 
+  def _create_delegated_target(self, rolename, keyids, threshold, paths):
+    """
+    Create a new Targets object for the 'rolename' delegation.  An initial
+    expiration is set (3 months from the current time).
+    """
+
+    expiration = tuf.formats.unix_timestamp_to_datetime(
+        int(time.time() + TARGETS_EXPIRATION))
+    expiration = expiration.isoformat() + 'Z'
```

Comment:
Unrelated to these changes, but does anybody know why `unix_timestamp_to_datetime` isn't including the `Z` suffix by default? Feels like an API error if the times are intended to always be GMT 🙂